src/main/java/service/impl/FruitBalanceCheckService.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;
+
+import java.util.Map;
+
+public class FruitBalanceCheckService {

```

Comment:
delete this class
you can check if the balance is negative in the mapper

Reply:
its fine, leave it